### PR TITLE
Change cursor when mouse hover over the location link

### PR DESCRIPTION
**Pull request by xamgore**
### tl;dr

before → after

### explained

On the page with detailed event description there is a block with links:
* link to the speaker's social network account
* link to the event in a social network (like vk meeting)
* link to map

But actually the last item isn't a link, it's just a button which triggers a modal window.
It also doesn't have an `href` attribute, so chrome doesn't change cursor's type (to pointer).
This commit fixes this strange behaviour. Some people (including me) might be confused with it.

The better way to solve the problem is to handle requests like `it61.info/events/some-talk?map`, but this requires some SPA stuff.

### CODE REVIEW

**Comment by xamgore** [comment]
Sorry, I don't understand the reasons why CI could fail :anguished:

**Comment by xamgore** [comment]
Is there anybody here? :octocat:
